modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -11547,6 +11547,23 @@ export const console: ConsoleNS = {
                             fileFormatSampleHeading: "Sample CSV file format:"
                         }
                     },
+                    inviteParentUserWizard: {
+                        totalInvitations: "Total Invitation(s)",
+                        successAlert: {
+                            description: "Successfully invited the user(s).",
+                            message: "Invitation(s) Sent"
+                        },
+                        errorAlert: {
+                            description: "An error occurred while inviting {{ failedCount }} user(s).",
+                            message: "Review Required"
+                        },
+                        tableMessages: {
+                            userNotFound: "User not found",
+                            activeInvitationExists: "An active invitation already exists for the user",
```

Comment:
Let's get the content reviewed by the doc team as well.

```suggestion
                            activeInvitationExists: "An active invitation for the user already exists",
```

Reply:
Changed